CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Ewon Flexy Extensions Library Changelog
```

Comment:
The version number needs to be bumped from 1.5.0 to 1.6.0 in the README.md file.

Reply:
@alexjhawk  what line ?  it is changed in line 65.  You have this indicated on CHANGELOG.md

Reply:
Yes, you got it at line 65. I added the comment here to associate with this commit, but couldn't comment on README.md as there we no changes to comment on.